### DISCUSSION TITLE
关于Win实现本地OCR的参考

### DISCUSSION THREAD

**Discussion by Ctrl-Ling** [shuax/MouseInc.Settings#383] [Ideas]
近期在微软官方工具[PowerToys](https://github.com/microsoft/PowerToys)中看到了新增OCR的功能，好奇研究了一下，发现其是基于Windows API实现的本地OCR，分享一下

官方文档：https://docs.microsoft.com/zh-cn/windows/powertoys/text-extractor
来源项目：https://github.com/TheJoeFin/Text-Grab

**Comment by shuax** [comment]
不会用C++调用

**Reply by Ctrl-Ling** [reply]
官方有个JS的调用demo https://github.com/microsoft/Windows-universal-samples/tree/main/archived/OCR
python也能调用 https://www.jianshu.com/p/9be58de78ded
当然只是仅供参考

**Comment by trhy631** [comment]
Microsoft Store里面无法下载该软件

**Reply by Ctrl-Ling** [reply]
https://github.com/microsoft/PowerToys/releases
